packages/cloud-cognitive/src/components/Datagrid/styles/datagrid.scss

```diff
@@ -264,6 +258,10 @@
   height: calc(100% + 40px);
 }
 
+.#{$block-class}__grid-container.#{$block-class}__with-pagination {
+  height: calc(100% - 40px);
```

Comment:
Again, think it’d be good to include a comment, but also we should use spacing tokens or rem here.

Reply:
Ths calc is for creating room for pagination under the table body. I'll include comments and make it rem value.